nautobot/extras/filters.py

```diff
@@ -734,10 +746,11 @@ class RelationshipAssociationFilterSet(BaseFilterSet):
     destination_type = ContentTypeMultipleChoiceFilter(
         choices=FeatureQuery("relationships").get_choices, conjoined=False
     )
+    peer_id = PeerIDFilter(field_name="peer_id")
```

Comment:
`peer_id` isn't a database field so `field_name` feels suspect to me?

Rather than implementing a custom filter class, I was expecting this would just be a `MultiValueUUIDFilter` with a custom `method` parameter implemented on the FilterSet - did that not work, or am I misremembering our best practices?

Reply:
I wasnt aware of MultiValueUUIDFilters and changed the code according to best practices.

Reply:
I'm wondering if for a future enhancement we could narrow the filter widget-side upon instantiation.